SecondHomework/Logistic/ParcelManager.cs

```diff
@@ -49,4 +58,31 @@ public Result RemoveParcel(Guid id, List<Parcel> parcelsList)
         }
         return result;
     }
-}
\ No newline at end of file
+
+    public Dictionary<WeightCategory, List<Parcel>> GetParcelsByWeight()
+    {
+        var parcelsByWeight = new Dictionary<WeightCategory, List<Parcel>>
+    {
+        { WeightCategory.UpTo1Kg, _parcels.Where(parcel => parcel.Weight <= 1).ToList() },
+        { WeightCategory.UpTo2Kg, _parcels.Where(parcel => parcel.Weight > 1 && parcel.Weight <= 2).ToList() },
+        { WeightCategory.UpTo5Kg, _parcels.Where(parcel => parcel.Weight > 2 && parcel.Weight <= 5).ToList() },
+        { WeightCategory.UpTo10Kg, _parcels.Where(parcel => parcel.Weight > 5 && parcel.Weight <= 10).ToList() },
+        { WeightCategory.UpTo20Kg, _parcels.Where(parcel => parcel.Weight > 10 && parcel.Weight <= 20).ToList() },
+        { WeightCategory.MoreThan20Kg, _parcels.Where(parcel => parcel.Weight > 20).ToList() }
+    };
+
+        return parcelsByWeight;
+    }
+    public string[] GetWeightCategories()
+    {
+        return Enum.GetValues(typeof(WeightCategory))
```

Comment:
I would suggest to use https://learn.microsoft.com/en-us/dotnet/api/system.enum.getnames?view=net-8.0